src/api/users/src/services/firestore.js

```diff
@@ -42,13 +42,14 @@ if (
       client_x509_cert_url: FIREBASE_CLIENT_X509_CERT_URL,
     }),
   });
-  logger.debug('Server running in online mode');
+  logger.info('Server running in online mode');
 } else {
+  process.env.FIRESTORE_EMULATOR_HOST = 'localhost:8088';
```

Comment:
Why are you overriding this?  It should be read into the env not set here.

Reply:
firestore.js has no access to env variables without dotenv requiring it in. This needs to be set somewhere in order for express to communicate with firebase when running in dev mode. It has to be manually set each time.

I couldn't get dev mode to properly work with Docker (talked about more in the PRs description) so this was my alternative solution for now.

Reply:
It needs to be set, but we'll set it when we start the service via Docker or tests.

You mean when you run `npm run dev` locally?

Reply:
>You mean when you run npm run dev locally?

Yeah exactly.

I set it in `development.yml` and confirmed it was being read via a console.log, but after running the users service in dev mode via Docker, it wasn't able to communicate with Firebase for some reason. I spent a while trying to get it to work but couldn't, I'm not sure what the issue is when running dev in Docker, hence this temp hack until this can be figured out.